chrome/content/zotero/xpcom/translation/translate.js

```diff
@@ -1528,7 +1535,8 @@ Zotero.Translate.Base.prototype = {
 		this._aborted = false;
 		this.saveQueue = [];
 		
-		Zotero.debug("Translate: Parsing code for "+translator.label, 4);
+		Zotero.debug("Translate: Parsing code for " + translator.label
+			+ " -- " + translator.translatorID + " -- " + translator.lastUpdated, 4);
```

Comment:
And the extra info really is parenthetical. Generally people will have the right translator and the right version.